src/pages/_app.js

```diff
@@ -0,0 +1,71 @@
+// @flow
+
+// babel-preset-react-app uses useBuiltIn "entry". We therefore need an entry
+// polyfill import to be replaced with polyfills we need for our targeted browsers.
+import '@babel/polyfill';
+
+import React from 'react';
+import BaseApp, { Container } from 'next/app';
+import Error from 'next/error';
+
+import GlobalStyle from '../GlobalStyle';
+import LeafletStyle from '../LeafletStyle';
+import AppStyle from '../AppStyle';
+import MapStyle from '../MapStyle';
+import {
+  loadExistingLocalizationByPreference,
+  expandedPreferredLocales,
+  parseAcceptLanguageString,
+} from '../lib/i18n';
+
+export default class App extends BaseApp {
```

Comment:
You're right. I have the feeling we might be able to get rid of big parts of the original Loader/App components, once the data fetching is moved in next.js page components. But this is something I would like to discuss with you all today. Especially because you have a much better overview of the project and how it might be used in the future.